dmt/main/models.py

```diff
@@ -1250,17 +1250,18 @@ def update_email(self, comment, user, skip_self=True):
             body
         )
         if skip_self:
-            addresses = [u.email for u in self.users_to_email(user)]
+            addresses = self.users_to_email(user)
         else:
-            addresses = [u.email for u in self.users_to_email()]
+            addresses = self.users_to_email()
 
         send_mail(clean_subject(email_subj), email_body, settings.SERVER_EMAIL,
                   addresses, fail_silently=settings.DEBUG)
         statsd.incr('main.email_sent')
 
     def users_to_email(self, skip=None):
+        """Returns a list of email addresses."""
         return [
-            n.user.userprofile
+            (n.user.userprofile.email or n.user.email)
```

Comment:
Can we pull this out as a method on `UserProfile` like `.get_email()` so we can re-use it in the other places in the PMT where we email the user?